Firestore/Example/Tests/SpecTests/FSTSyncEngineTestDriver.mm

```diff
@@ -199,16 +201,14 @@ - (instancetype)initWithPersistence:(id<FSTPersistence>)persistence
                                                  std::make_shared<EmptyCredentialsProvider>());
     _remoteStore = absl::make_unique<RemoteStore>(
         _localStore, _datastore, _workerQueue, [self](OnlineState onlineState) {
-          [self.syncEngine applyChangedOnlineState:onlineState];
-          _eventManager->HandleOnlineStateChange(onlineState);
+          _syncEngine->HandleOnlineStateChange(onlineState);
+          // _eventManager->HandleOnlineStateChange(onlineState);
```

Comment:
Commented-out code.

Reply:
Discussed offline -- it's redundant because the line above calls that already. Great catch; please port this to other platforms as well.